Example/Example/Shared/Dependencies.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Dependencies.swift
+//  Example
+//
+//  Created by Adam Campbell on 22/7/20.
+//  Copyright © 2020 Afterpay. All rights reserved.
+//
+
+import Afterpay
+import CommonCrypto
+import Foundation
+import TrustKit
+
+func initializeDependencies() {
+  // In a real world scenario this would be a real backup hash but for demonstration purposes
+  // it is an empty hash to satisfy TrustKit's requirements
+  let backupHash = Data(count: Int(CC_SHA256_DIGEST_LENGTH)).base64EncodedString()
+
+  let configuration: [String: Any] = [
+    kTSKSwizzleNetworkDelegates: false,
+    kTSKPinnedDomains: [
+      "portal.afterpay.com": [
+        kTSKExpirationDate: "2022-06-25",
```

Comment:
Just for my own benefit—will pinning automatically fail after this date or will it be disabled?

Adding an expiration date to a PIN in the Android network security configuration will bypass certificate pinning after that date.

Reply:
It will be disabled. Essentially downgrading the security after that date: https://datatheorem.github.io/TrustKit/documentation/Domain%20Configuration%20Keys.html#/c:@kTSKExpirationDate

The dates match the current expiry dates of certs that those hashes match

Reply:
Cool, that's what I thought and is the same behaviour as Android. 👍